src/Elastic.Apm/BackendComm/CentralConfig/CentralConfigurationResponseParser.cs

```diff
@@ -74,6 +74,19 @@ private static CentralConfigurationFetcher.WaitInfoS ExtractWaitInfo(HttpRespons
 		{
 			if (httpResponse.Headers?.CacheControl?.MaxAge != null)
 			{
+				if (httpResponse.Headers.CacheControl.MaxAge <= TimeSpan.FromSeconds(0))
+				{
+					return new CentralConfigurationFetcher.WaitInfoS(WaitTimeIfNoCacheControlMaxAge,
+						"The max-age directive in Cache-Control header in APM Server's response is zero or negative, "
```

Comment:
The wording is maybe a bit cumbersome but I did not come up with a real improvement, so let's just leave it as it is 😄 .